service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/base/AbstractEmailContentProvider.java

```diff
@@ -26,6 +27,10 @@ public String getCaseUrl(Long caseId) {
         return caseUrlService.getCaseUrl(caseId);
     }
 
+    public String getCaseUrl(CaseData caseData) {
```

Comment:
Can't find where this is being used. Can we remove if not needed?

Reply:
removed it